.cirrus.yml

```diff
@@ -178,6 +176,8 @@ task:
 
 task:
   name: 'macOS 10.14 [gui, no tests] [focal]'
```

Comment:
why only under macOS 10.14, shouldn't it be added to the macOS 11 task as well?

Reply:
Jup it is "native" (an actual mac compiling), not a cross-compile with depends